src/magic_folder/api_cli.py

```diff
@@ -106,14 +108,20 @@ def dump_state(options):
             q.extend(s.parents_local)
         print("    {}: {}".format(snap_name, parents[:-4]), file=options.stdout)
     print("  remote snapshots:", file=options.stdout)
-    for snap_name in config.get_all_snapshot_paths():
+    for snap_name, ps, last_update, upload_duration in config.get_all_current_snapshot_pathstates():
         try:
             cap = config.get_remotesnapshot(snap_name)
         except KeyError:
             cap = None
-        path_state = config.get_currentsnapshot_pathstate(snap_name)
-        print("    {}: {}".format(snap_name, cap), file=options.stdout)
-        print("    {}  {}".format(" "*len(snap_name), path_state), file=options.stdout)
+            continue
+        print("    {}:".format(snap_name), file=options.stdout)
+        print("        cap: {}".format(cap), file=options.stdout)
+        print("        mtime: {}".format(ps.mtime_ns), file=options.stdout)
+        print("        size: {}".format(ps.size), file=options.stdout)
+        if upload_duration:
+            duration = upload_duration / 1000000000.0  # humans care about seconds not ns
```

Comment:
We have a function for this.

Reply:
I didn't use it here because it truncates to "seconds" .. and I figured this is one case we probably _do_ want fractional seconds so that speed calculations (esp. for smaller files) are more accurate. Maybe _all_ seconds want fractional parts, though, and `ns_to_seconds` should just return float?

Reply:
Hmm. I think all the places we currently use it, second accuracy is reasonable to have; which inclines me to not change the current interface. Can you leave a comment here that explains why you aren't using the helper?

Reply:
made a new helper with a docstring